lib/qless/server/views/layout.erb

```diff
@@ -13,7 +13,7 @@
     <link href="<%= u '/css/jquery.noty.css' %>" rel="stylesheet">
     <link href="<%= u '/css/noty_theme_twitter.css' %>" rel="stylesheet">
     <link href="<%= u '/css/failed-jobs-list.css' %>" rel="stylesheet">
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
Somewhat unrelated, but is this a good opportunity to bundle jQuery with Qless in our own fork? As it stands, our CSP has to include `ajax.googleapis.com` and I'd much prefer having no third party scripts at all.